### DISCUSSION TITLE
How to modify the code so as not to lose data

### DISCUSSION THREAD

**Discussion by cdhigh** [OpenHantek/OpenHantek6022#201] [Q&A] [answered]
I found that sometimes the Oscilloscope cannot trigger with single trigger actived.
With some search I found It is a limit of hardware.
And The software also considers not occupying too much cpu.
This will cause a lot of data can not be transfered to computer.
I haver a old laptop for 6022 only, the laptop will not run other software when 6022 is running.
So, 100% of cpu ocupation is not a problem for me (and for laptop).
How can I modify the source to achieve it?
Thanks in advanced.

**Comment by Ho-Ro** [comment, marked as answer]
You can play with [this setting](https://github.com/OpenHantek/OpenHantek6022/blob/d3a99690fe2b5cd5fddade2eebc2b927a7f43033/openhantek/src/scopesettings.h#L35):
`    double acquireInterval = 0.001; ///< Minimal time between captured frames`
Values < 0.001 will result in no extra delay, so just put in e.g. 0.0001

**Reply by cdhigh** [reply]
Thanks for answer
